public/components/freelancer-edit.js

```diff
@@ -535,9 +539,247 @@ function renderCalendar() {
       }
     }
   });
+}
+
+function updateForms(freelancer){
+	freelance = freelancer;
+
+	document.getElementById('email').value = freelancer.email;
+	document.getElementById('description').value = freelancer.description;
+	let temp = "";
+	for( let tag of freelancer.tags){
+		temp+=tag.tagName+",";
```

Comment:
I'd add a space here. Also in the creation form and in the router you should `split` on `", "`.